EIPS/eip-7580.md

```diff
@@ -0,0 +1,57 @@
+---
+eip: 7580
```

Comment:
```suggestion
eip: unassigned
```
A number hasn't been assigned yet.  Please move this to the ERC repo.

Reply:
Oops. Got it